core/src/main/resources/jenkins/agents/CloudSet/index.jelly

```diff
@@ -39,22 +39,47 @@ THE SOFTWARE.
               </a>
             </l:isAdmin>
           </l:app-bar>
-          <table id="clouds" class="jenkins-table sortable">
+          <table id="clouds" class="jenkins-table">
             <thead>
               <tr>
-                <th class="jenkins-table__cell--tight"/>
-                <th initialSortDir="down">${%Name}</th>
-                <th class="jenkins-table__cell--tight"/>
+                <th/>
+                <th colspan="2">${%Order}</th>
+                <th>${%Name}</th>
+                <th>${%Configure}</th>
               </tr>
             </thead>
             <tbody>
-              <j:forEach var="cloud" items="${it.clouds}">
+              <j:forEach var="cloud" items="${it.clouds}" varStatus="status">
                 <tr id="cloud_${cloud.name}">
                   <td data="${cloud.icon}" class="jenkins-table__cell--tight jenkins-table__icon">
                     <div class="jenkins-table__cell__button-wrapper">
                       <l:icon src="${cloud.iconClassName}" tooltip="${cloud.iconAltText}"/>
                     </div>
                   </td>
+                  <td class="jenkins-table__cell--tight">
+                    <div class="jenkins-table__cell__button-wrapper">
+                      <f:form method="post" action="move" name="config" class="jenkins-form">
+                        <a href="move" class="jenkins-table__button">
+                          <input type="hidden" name="name" value="${cloud.name}"/>
+                          <j:if test="${not status.first}">
+                            <f:submit name="up" value="" icon="symbol-chevron-up"/>
```

Comment:
I think a `tertiary` or button with a symbol would look less intrusive, especially if you have multiple clouds set up: https://weekly.ci.jenkins.io/design-library/Buttons/

Reply:
I updated the PR with tertiary buttons. Let me know what you think!

Reply:
Thanks!